playbooks/prepare_host.yaml

```diff
@@ -37,14 +37,19 @@
         - name: Configure Red Hat Subscription Manager
           ansible.builtin.import_role:
             name: redhat-subscription
+        # this needs to be done early
+        - name: set rhsm_release
+          shell: subscription-manager release --set={{rhsm_release}}
```

Comment:
Let's not do that. We have an Ansible role to manage RHSM, the release should be set by the role already with `rhsm_release`.